scania-digital-design-system/sdds
# feat(card): webcomponent

**Describe pull-request**  
Web component version of card. Also updated the variable for card body color.

**Solving issue**  
Fixes: [AB#3147](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/3147)

**How to test**  
1. Go to storybook link below.
2. Check in Components -> Card -> WebComponent
3. Try out the controls and check that the design is correct.

**Suggested test steps**

**Screenshots**  


**Additional context**
